### ISSUE TITLE
[Question] How do I specify the assembly name when using external properties config

### ISSUE THREAD

**Issue by pn-santos** [fabric8io/docker-maven-plugin#974]
### Description

I'm using an external image (property-based) config with an assembly descriptor file, however I can't seem to be able to change the default `/maven` dir name (to where the assembly files are copied to)... There doesn't seem to be an `assembly.name` property. I tried setting `docker.assembly.baseDir` but that doesn't seem to have any effect (judging by the docs it should set that dir shouldn't it?)

### Info

* d-m-p version : 0.24.0
* Maven version (`mvn -v`) :
```
Apache Maven 3.5.2 (138edd61fd100ec658bfa2d307c43b76940a5d7d; 2017-10-18T08:58:13+01:00)
Maven home: /opt/maven/latest
Java version: 1.8.0_151, vendor: Oracle Corporation
Java home: /opt/java/jdk1.8.0_151/jre
Default locale: en_GB, platform encoding: UTF-8
OS name: "linux", version: "4.13.0-36-generic", arch: "amd64", family: "unix"
```
* Docker version : 17.12.1-ce

**Comment by rohanKanojia** [comment]
Hi, Did you get any chance to read [this](https://dmp.fabric8.io/#build-assembly)?

**Comment by pn-santos** [comment]
Yes, when I configure an inline assembly it's easy, I just set the assembly name and that will be the name where the assembly files are copied to. That works fine. 

However when using *[external properties configuration](https://dmp.fabric8.io/#property-configuration)* I cannot find the equivalent property.

As I mentioned in the OP the docs for `assembly.baseDir` say:
> _Directory name for the exported artifacts as described in an assembly (which is /maven by default)._

my interpretation is that this property should be the equivalent of the inline assembly name property. But when I set it, it seems to have no effect whatsoever (I still get the `/maven` dir independently of the value of that property).

**Comment by rhuss** [comment]
Actually, you should set the property `docker.assembly.baseDir` to set the name (or another prefix instead of `docker` if you have changed it like described in https://dmp.fabric8.io/#property-configuration

Have you used the full property name ?

If you did, and it didn't work for you, could you please post a quick example where you see this issue ?

btw, actually it should be called `targetDir` (as it is called in the assembly config) not `baseDir`, but the property mechanism doesnt seem have to be updated properly when this has changed.

**Comment by pn-santos** [comment]
I'll try to post a quick example project (the current one I can't post). Not sure if relevant but I'm using a custom prefix (`<prefix>docker.service-image</prefix>`) not the default one, which means I'm setting `docker.service-image.assembly.baseDir`.

**Comment by pn-santos** [comment]
@rhuss here's the example: https://github.com/pn-santos/docker-maven-example

I've set `<docker.image.assembly.baseDir>assets</docker.image.assembly.baseDir>` however I still get `maven` as the build dir

**Comment by pn-santos** [comment]
@rhuss did you get the chance to try out the example project?
